python/tvm/relay/frontend/onnx.py

```diff
@@ -3794,12 +3850,14 @@ def _get_convert_map(opset):
         "Xor": Renamer("logical_xor"),
         # defs/nn
         "AveragePool": AveragePool.get_converter(opset),
+        "QLinearAveragePool": QLinearAveragePool.get_converter(opset),
```

Comment:
There's a quantization section down below, you should move these there

Reply:
Thanks for the review, I've made the change.